src/App.js

```diff
@@ -130,6 +131,7 @@ const App = ({ locale, location, dependencies: { appSessionRef, sessionManager }
                 component={ReportsPartialsCampaigns}
               />
               <PrivateRoute path={['/billing/invoices']} exact component={InvoicesList} />
+              <PrivateRoute path={['/integrations/data-studio/free']} exact component={FreeAccount} />
```

Comment:
No need to use path= {['/integrations/data-studio/free']}, you could do it like this  path= '/integrations/data-studio/free'

Reply:
sueggest acepted and applied !